test/libsolidity/smtCheckerTests/inheritance/state_variables_2.sol

```diff
@@ -2,12 +2,10 @@ pragma experimental SMTChecker;
 
 contract Base1 {
 	uint x;
-	uint private t;
 }
 
 contract Base2 is Base1 {
 	uint z;
-	uint private t;
```

Comment:
Why did you remove it? Would it cause an error?

Reply:
It wouldn't but it wasn't being used. Probably copied from some other test and forgot then.